### ISSUE TITLE
Add generator for views and integrate it on controllers generator

### ISSUE THREAD

**Issue by roman-dubrovsky** [datarockets/rails-views#3]

**Comment by AleksSenkou** [comment]
@roman-dubrovsky I'd say it has super low priority because we don't use scaffold

Let me know if I'm wrong // @DDKatch @nikitasakov

**Comment by DDKatch** [comment]
@AleksSenkou  I don't think so, It's boring to create at least 2 files and 1 folder every time.

**Comment by AleksSenkou** [comment]
@DDKatch I got your point. Please, remove `postponed` label if you feel that the feature has average priority 😉
